<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifResaDialog</class>
 <widget class="QDialog" name="ModifResaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>466</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Informations</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="text">
              <string>(Nom Hotel - Numéro de Chambre - Nombre de Couchages - Prix - Adresse)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item alignment="Qt::AlignVCenter">
            <widget class="QComboBox" name="comboBox_listeHotel"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="logo">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>logoProjet_petit_petit.png</pixmap>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QPushButton" name="accueil">
           <property name="text">
            <string>Accueil</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>MODIFICATION DE RESERVATION</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item alignment="Qt::AlignVCenter">
          <widget class="QPushButton" name="pushButton_val">
           <property name="text">
            <string>VALIDER</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Date de Départ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_dateDep">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Date d'Arrivée</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_dateArr"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Mail Client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_mailCli">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
